src/main/flight/pid.c

```diff
@@ -883,7 +883,7 @@ NOINLINE static void calculateSpaValues(const pidProfile_t *pidProfile)
     }    
 #else
     UNUSED(pidProfile);
-#endif // #ifdef USE_WING ... #else
+#endif // USE_WING ... #else
```

Comment:
No need to include #else :)
```suggestion
#endif // USE_WING
```